packages/emails/src/templates/VerifyAccountEmail.tsx

```diff
@@ -44,13 +44,13 @@ export const VerifyAccountEmail = (
       <div style={{ lineHeight: "6px" }}>
         <p style={{ fontWeight: 400, lineHeight: "24px" }}>
           <>
-            {props.language("happy_scheduling")} <br />
+            {props.language("happy_scheduling")}, <br />
             <a
               href={`mailto:${SUPPORT_MAIL_ADDRESS}`}
               style={{ color: "#3E3E3E" }}
               target="_blank"
               rel="noreferrer">
-              <>{props.language("the_calcom_team", { companyName: COMPANY_NAME })}</>
+              <>{props.language("the_calcom_team", { companyName: SENDER_NAME })}</>
```

Comment:
Can this be a breaking change?(for self hosters). I mean what if this env variable isn't set?

Reply:
Sure i will push these changes ... very good eyes @hariombalhara